application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3328,6 +3345,7 @@ enum RelativeDirection {
   CONTINUE
   DEPART
   ELEVATOR
+  ENTER_OR_EXIT_STATION
```

Comment:
We don't want to expose this enum value to users, it should be mapped to CONTINUE.